connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -193,11 +193,32 @@ public void testCreateConnectorExists() throws Throwable {
         PowerMock.verifyAll();
     }
 
+    // Requests to create a connector that do not contain a name in either the body or the config object should be
+    // rejected outright
     @Test(expected = BadRequestException.class)
-    public void testCreateConnectorWithASlashInItsName() throws Throwable {
-        String badConnectorName = CONNECTOR_NAME + "/" + "test";
+    public void testCreateConnectorWithoutName() throws Throwable {
```

Comment:
This is an empty name, not null, right? Maybe clarify, since the difference is meaningful.

Reply:
Wouldn't it also be useful to keep the `testCreateConnectorWithInvalidName()` while also adding `testCreateConnectorWithoutName()`?